File: NewFantasyUI/src/app/model/modifier-update-request.ts
export class ModifierUpdateRequest {
	
	userId: string;
	modifierId: string;
	playerId: string;
	action: string;
	
	constructor(userId: string, modifierId: string, playerId: string, action: string) { 
		this.userId = userId;
		this.modifierId = modifierId;
		this.playerId = playerId;
		this.action = action; 
	}
}

File: DataAccessUtility/src/main/java/com/fantasy/dataaccessutility/model/ui/ModifierUpdateRequest.java
package com.fantasy.dataaccessutility.model.ui;

public class ModifierUpdateRequest {
	
	private String userId;
	private String modifierId;
	private String playerId;
	private String action;
	
	public String getUserId() {
		return userId;
	}
	public void setUserId(String userId) {
		this.userId = userId;
	}
	public String getModifierId() {
		return modifierId;
	}
	public void setModifierId(String modifierId) {
		this.modifierId = modifierId;
	}
	public String getAction() {
		return action;
	}
	public void setAction(String action) {
		this.action = action;
	}
	public String getPlayerId() {
		return playerId;
	}
	public void setPlayerId(String playerId) {
		this.playerId = playerId;
	}
	
	

}
